packages/page/src/Application/MessageHandler/RemovePageMessageHandler.php

```diff
@@ -29,8 +30,8 @@ final class RemovePageMessageHandler
     public function __construct(
         private PageRepositoryInterface $pageRepository,
         private DomainEventCollectorInterface $domainEventCollector,
+        private TrashManagerInterface $trashManager
```

Comment:
Trash should stay optional:

```suggestion
        private ?TrashManagerInterface $trashManager = null,
```